docs/ProductSpecifications.md

```diff
@@ -1,68 +1,63 @@
-# Product Specifications
+📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion). 
 
-## Description
+# Product Specifications
 
 `ProductSpecifications` is a VTEX component that shows the technical specifications of a product.
```

Comment:
`ProductSpecifications` is a block that shows the technical [specifications](https://help.vtex.com/tutorial/what-are-fields-or-specifications--2lB4AgibEseceMggKE2k2m) of a product and its SKUs.